examples/frompapers/Izhikevich_2003.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+##Author: Akif Erdem Sağtekin
+
+##Implemented from "https://www.izhikevich.org/publications/spikes.htm"
+
+############################
+
+# I am in favor of coding simplicity for scientific computing when applicable.
+# Since there are few neural groups and connections in this simulation, 
+# all the neuron groups and synaptic connections are explicitly coded line by line, 
+# without even using any for loops. 
+
+############################
+
+from brian2 import *
+from brian2tools import *
+import numpy as np
+np.random.seed(1337)
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+defaultclock.dt = 0.5*ms 
+
+# izhikevich model.
+
+eqs = ''' dv/dt = (0.04/ms/mV)*v**2 + (5/ms)*v + 140*mV/ms - u/ms + I*mV/(amp*ms) : volt
+          du/dt = a*(b*v - u)/ms  : volt
+          I : amp
+          a : 1
+          b : 1
+          c : volt
+          d : volt '''
+
+# synaptic parameters are fine-tuned in order to see the activity in the paper.
+# in paper, they used -1*amp for inhibitory and 0.5*amp for excitatory.
```

Comment:
do you know why the change is necessary? are the numbers in the paper a typo? is the model different?

Reply:
I don't know unfortunately. I experimentally find out that multiplying these two weights with 4 gives more similar results with MATLAB code of the paper. Maybe specifying the noise as you suggest will fix it,so just waiting for your response about noise.